govc/library/import.go

```diff
@@ -158,6 +169,15 @@ func (cmd *item) Run(ctx context.Context, f *flag.FlagSet) error {
 			LibraryItemID: cmd.ID,
 		})
 
+		if cmd.pull {
+			_, err = m.AddLibraryItemFileFromURI(ctx, session, filepath.Base(file), file)
+			if err != nil {
+				return err
+			}
+
+			return m.WaitOnLibraryItemUpdateSession(ctx, session, 3*time.Second, nil)
```

Comment:
Hi @dougm,

While you *can* use the helper function to wait, I wonder if you want to implement the progress sink here since?

Reply:
Thanks for the review @akutz .  Good idea, `Session` does have a `ClientProgress` we can use for this.  Though we also need the `error` wrapper for `State` for the `progress.Report.Error()` impl.  So I'll merge this as-is, and see about including progress here as part of the error related changes we discussed.

Reply:
sgtm, thanks again!